test/__snapshots__/static.js.snap

```diff
@@ -142,7 +142,22 @@ exports[`static rules do static things 1`] = `
 .mix-blend-initial{mix-blend:initial;}
 .mix-blend-revert{mix-blend:revert;}
 .mix-blend-revert-layer{mix-blend:revert-layer;}
-.mix-blend-unset{mix-blend:unset;}"
+.mix-blend-unset{mix-blend:unset;}
+.touch-auto{touch-action:auto;}
+.touch-none{touch-action:none;}
+.touch-pan-x{touch-action:pan-x;}
+.touch-pan-y{touch-action:pan-y;}
+.touch-pan-left{touch-action:pan-left;}
+.touch-pan-right{touch-action:pan-right;}
+.touch-pan-up{touch-action:pan-up;}
+.touch-pan-down{touch-action:pan-down;}
+.touch-pinch-zoom{touch-action:pinch-zoom;}
+.touch-manipulation{touch-action:manipulation;}
+.touch-inherit{touch-action:inherit;}
+.touch-initial{touch-action:initial;}
+.touch-revert{touch-action:revert;}
+.touch-revert-layer{touch-action:revert-layer;}
+.touch-unset{touch-action:unset;}"
```

Comment:
Just curious, I can't see those [here](https://warp-ds.github.io/css-docs/touch-action)  Do we really need them? I've checked in tw as well and no such classes there either.

Reply:
<img width="555" alt="image" src="https://github.com/warp-ds/drive/assets/37986637/7f3c9d3d-b3b1-4dd1-95fd-045a59b29aa0">

Reply:
I don't know. Just feel that if they are not in tailwind I don't know why we should have them